webapp/package.json

```diff
@@ -25,7 +25,7 @@
     "localforage": "1.5.0",
     "marked": "mattermost/marked#5194fc037b35036910c6542b04bb471fe56b27a9",
     "match-at": "0.1.0",
-    "mattermost-redux": "mattermost/mattermost-redux#webapp-4.1",
+    "mattermost-redux": "ccbrown/mattermost-redux#PLT-6403",
```

Comment:
This will need to be updated to latest master once it's merged into mattermost-redux